app/include/zmk/keymap.h

```diff
@@ -27,6 +28,9 @@ const char *zmk_keymap_layer_name(uint8_t layer);
 int zmk_keymap_position_state_changed(uint8_t source, uint32_t position, bool pressed,
                                       int64_t timestamp);
 
+int zmk_invoke_behavior_binding(struct zmk_behavior_binding *binding,
```

Comment:
Not delved deep enough into the code to know the stylings typically used, but would it make sense to name this `invoke_zmk_behavior_binding`?

Reply:
My impression (which can also be wrong) is that exposed methods (declared in the headers) get the `zmk_` prefix.